orders/tests.py

```diff
@@ -0,0 +1,61 @@
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import Order
+from .models import StatusOptions
+from django.core.exceptions import ObjectDoesNotExist
+import pytest
+
+STARTING_DATE = timezone.now()
+FINISHING_DATE = timezone.now()
+TOTAL_PRICE = 100
+STATUS = StatusOptions.Pending
+
+
+@pytest.fixture
+def order():
+    return Order.create(dog_owner_id=DogOwner.objects.get(id=1),
+                        daycare_id=DayCare.objects.get(id=1),
+                        starting_date=STARTING_DATE,
+                        finishing_date=FINISHING_DATE,
+                        total_price=TOTAL_PRICE,
+                        )
+
+
+@pytest.mark.django_db
+class TestOrderModel:
+    def test_create_order(self):
+        new_order = Order.create(dog_owner_id=DogOwner.objects.get(id=1),
+                                 daycare_id=DayCare.objects.get(id=1),
+                                 starting_date=STARTING_DATE,
+                                 finishing_date=FINISHING_DATE,
+                                 total_price=TOTAL_PRICE)
+        assert new_order in Order.objects.all()
+
+    def test_delete_order(self, order):
+        assert order in Order.objects.all()
+        order.delete()
+        assert order not in Order.objects.all()
+
+    def test_get_not_existed_order_by_order_id(self):
+        try:
+            Order.objects.get(id=-1)
+            assert False
+        except ObjectDoesNotExist:
+            assert True
+
+    def test_approve_order(self, order):
+
+        order.approve_order()
+        order.refresh_from_db()
```

Comment:
Why is it necessary to call refresh_from_db()?

Reply:
Because the order instance before 

> Hi Ofir, Please fix the commit message body, it should include at least what described by the PR description.
> 
> This PR describes that it will be possible to see the orders history, by it seems that once a daycare **or** a dogowner user is deleted the order is deleted as well, meaning the history is lost 🙂 Are you going to address it in this PR?

I'll change the commit, but I've mentioned my opinion about deletion of a user and its affect on orders in another conversation here :)

Reply:
> Why is it necessary to call refresh_from_db()?

It not. I've used it to check if it's the cause of a failing test, but it wasn't.
Removed.